### PR TITLE
Add javascript_pack_tag to README.md [ci skip]

### PR DESCRIPTION
**Pull request by olimart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,16 @@ Then, render it in your views with the `component` helper (or its alias `c`).
 = c "button"
 ```
 
+Make sure to include javascript pack tag in your application layout file, for instance:
+
+```slim
+/ app/views/layouts/application.html.erb
```
Comment (florentferry) [inline README.md]: File extension is `slim` to keep consistency between examples in readme.

File: README.md [hunk 2]
```diff
@@ -71,6 +71,16 @@ Then, render it in your views with the `component` helper (or its alias `c`).
...
+
+= javascript_pack_tag "application"
```
Comment (florentferry) [inline README.md]: A full example with `stylesheet_pack_tag` in the head of your layout, and `javascript_pack_tag` before the closing `body` tag should be better here.

**Comment by olimart** [comment]
@florentferry Let me know if I got the syntax wrong as I'm not used to it.

File: README.md [hunk 3]
```diff
@@ -71,6 +71,19 @@ Then, render it in your views with the `component` helper (or its alias `c`).
...
 
+Make sure to include javascript pack tag and stylesheet pack tag in your application layout file, for instance:
+
+```slim
+/ app/views/layouts/application.html.slim
+
+body=
+  = yield
+  = javascript_pack_tag "application"
+  = stylesheet_pack_tag "application"
```
Comment (Spone) [inline README.md]: Please put the `stylesheet_pack_tag` in the `head` and add the doctype. See here for an example of slim syntax: http://slim-lang.com

**Pull request merged**